pkg/virt-controller/watch/vmi_test.go

```diff
@@ -1605,6 +1607,59 @@ var _ = Describe("VirtualMachineInstance watcher", func() {
 
 			controller.Execute()
 		})
+		Context("And the virt-launcher pod has Multus annotations and SRIOV interface", func() {
+			DescribeTable("should copy Multus annotations to VirtualMachineInstance and change vmi status to Scheduled", func(oldStatusAnnotation string) {
+				By("creating a vmi in Scheduling phase and a SRIOV interface")
+				vmi := NewPendingVirtualMachine("testvmi")
+				setReadyCondition(vmi, k8sv1.ConditionFalse, virtv1.PodConditionMissingReason)
+				vmi.Status.Phase = virtv1.Scheduling
+				vmi.Spec.Domain.Devices.Interfaces = []virtv1.Interface{libvmi.InterfaceDeviceWithSRIOVBinding("sriov-interface")}
+				if oldStatusAnnotation != "" {
+					vmi.Annotations[networkv1.NetworkStatusAnnot] = oldStatusAnnotation
+				}
+
+				By("attaching a virt-launcher pod with Multus annotations to the vmi")
+				networkStatusAnnotation := `new-network-status`
```

Comment:
nit: Why is `new-network-status` surrounded by back-ticks?

Reply:
I initially used the full annotation that has " in it, but now that I'm using a dummy content it is not needed. Done